nullaway/src/main/java/com/uber/nullaway/fixserialization/FixSerializationConfig.java

```diff
@@ -119,7 +123,27 @@ public FixSerializationConfig(String configFilePath) {
         XMLUtil.getValueFromAttribute(
                 document, "/serialization/fieldInitInfo", "active", Boolean.class)
             .orElse(false);
-    serializer = new Serializer(this);
+    // Serialization version, if no version is defined, we will work with the latest version.
+    int serializationVersion =
+        XMLUtil.getValueFromTag(document, "/serialization/version", Integer.class)
+            // should always be the most recent version.
+            .orElse(2);
```

Comment:
Let's create a final static constant `LATEST_VERSION` in `SerializationAdapter`, and then replace `2` with `LATEST_VERSION` here and wherever else we are using `2` to mean the latest version

Reply:
e15a83b66d593e3e046441151312aaadeb2450fc